Source/ThirdParty/ANGLE/src/libANGLE/renderer/metal/mtl_command_buffer.mm

```diff
@@ -1768,12 +1800,22 @@ inline void CheckPrimitiveType(MTLPrimitiveType primitiveType)
     return *this;
 }
 
-RenderCommandEncoder &RenderCommandEncoder::setScissorRect(const MTLScissorRect &rect)
+RenderCommandEncoder &RenderCommandEncoder::setScissorRect(const MTLScissorRect &rect,
+                                                           id<MTLRasterizationRateMap> map)
 {
+    auto maxScissorRect =
+        MTLCoordinate2DMake(mRenderPassMaxScissorRect.width, mRenderPassMaxScissorRect.height);
+    if (!shouldUseRasterizationRateMap(mCachedRenderPassDescObjC))
```

Comment:
This in an addition from Dan's changes @kkinnunen-apple, since the scissor modification should only be applied on the double-width shared layout render target